utils/markdown/remove_markdown.js

```diff
@@ -5,31 +5,31 @@ import marked from 'marked';
 
 export default class RemoveMarkdown extends marked.Renderer {
     code(text) {
-        return ' ' + text.split('\n').join(' ');
+        return text.split('\n')[0];
```

Comment:
I don't think all these splits in the renderer should be necessary when you're doing it in `stripMarkdown` after this has all been done